test/Microsoft.Framework.Logging.Test/LoggerFactoryExtensionsTest.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Framework.Logging.Test
     public class LoggerFactoryExtensionsTest
     {
 #if ASPNET50
```

Comment:
Ah, yeah, the main logging test project shouldn't be depending on Serilog. Can you log a bug to split that out into a separate test project and have *that* one be `aspnet50`/`net45` (or whatever), and then change the main logging test project to run on everything? And then fix it? :smile: